Google.Api.Gax.Grpc/ChannelPool.cs

```diff
@@ -25,6 +25,8 @@ public sealed class ChannelPool
     {
         private readonly IEnumerable<string> _scopes;
 
+        internal readonly bool _useJwtAccessWithScopes = false;
```

Comment:
+1 for adding a constructor overload. (The current constructor can just delegate to that one.)

Reply:
@amanda-tarafa Ok, I agree that property would fit better, but why we want to make Jwt flag configurable? Could you please explain the scenario in mind? 

As I understand, right now Pools does not support Jwt because its not part of the key. I thought we better add constructor overload along with the update to the Pool implementation.

Reply:
Just for the record @TimurSadykov and I discussed this offline:

- Currently each pool either supports JWTs or not.
- If we, at some point, make the credential config part of the key on which the channels are pooled, then the same pool will be able to serve channels using JWTs and channels that don't use JWTs.

Reply:
synched offline, fixed